build/helper.py

```diff
@@ -711,7 +726,19 @@ def get_python_type_from_visa_type(visa_type):
     else:
         p_types = importlib.import_module('build.templates.python_types')
     v_type = getattr(p_types, visa_type)
-    p_type = v_type().python_type()
+    # If we just take type() and make it a string, we get <class 'float'> which is not what we want
```

Comment:
Changed. Actually __name__

Also renamed function to be explicit about returning the underlying intrinsic (basic python) type